dotnet-desktop-guide/framework/winforms/controls/how-to-designate-a-windows-forms-button-as-the-accept-button.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Designate a Button as the Accept Button
+description: Learn how to assign a button as the accept button also known as a default button on any windows form. 
```

Comment:
Learn how to assign a button as the accept button, also known as a default button, on any windows form.